spec/reviews_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+FactoryBot.define do
+  factory :restaurant do
+  end
+end
+
+
+describe  ReviewsController , :type => :controller do
+  context "GET index" do
+    it "has a 200 status code" do
+      get :index
+      expect(response.status).to eq(200)
+      expect(response.content_type).to eq "application/json"
+    end
+  end
+
+  context 'POST a new review' do
+    let(:restaurant) { FactoryBot.create(:restaurant)}
+    let(:rating) {3}
+
+    it "create a new review" do
```

Comment:
The test's title claims to verify a review was _created_, but claim isn't reflected at the code (Or am I missing it?)
I'd expected a new review to be persisted (i.e `expect(Review.last.rating).to eq(:rating)`)